examples/platformvm/importTx-avax-to-the-P-Chain-from-the-C-Chain-and-consume-a-multisig-atomic-output-and-a-create-multisig-output.ts

```diff
@@ -0,0 +1,117 @@
+import { Avalanche, BinTools, BN, Buffer } from "avalanche/dist"
```

Comment:
I think we have this example already with a shorter filename.
Pls. check, and if not InitAvalanche() pls.

Reply:
there's this `examples/evm/importTx-avax-to-the-pchain-and-consume-a-multisig-output.ts` which however is wrongly named since it imports to cchain from pchain. So I will rename it and keep also avax's example and refactor it.